webpack.shared.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
 				loader: 'babel-loader',
 				exclude: /node_modules/,
 			},
+			FileConfig.loader(),
```

Comment:
We should provide some `file-loader` config here so that the images are put into a public directory and the server can find them.

I've been unable to get that working correctly. They always seem to be written to `build/images`.